docs/source/getting-started/index.rst

```diff
@@ -0,0 +1,96 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _new-quickstart:
+
+Full Stack DAML
+###############
+
+**Disclaimer:** This guide is being actively developed.
+Expect major changes to the tutorial text and minor changes to the template application.
+
+The goal of this tutorial is to get you up and running with full-stack, DAML-driven app development. We will provide you with the template for a miniature social networking app, get you writing your first end-to-end feature and finally deploy your new app to a persistent ledger.
+
+By the end of the tutorial, you should have an idea of what DAML contracts and ledgers are, how the UI interacts with them, and how you might solve a potential use case with a DAML-driven solution. We do not aim to give a comprehensive guide to all DAML concepts and tools; for that, see the later sections of the documentation. With that, let's get started!
+
+.. TODO: reference specific sections of docs instead of saying "later sections".
+
+.. toctree::
+   :hidden:
+
+   app-architecture
+   first-feature
+
+Prerequisites
+*************
+
+If you haven't already, see :doc:`installation` for the DAML SDK and VSCode development environment.
+
+You will also need some common software tools to build and interact with the template project.
+
+- `Git <https://git-scm.com/>`_ version control system
+- `Yarn <https://yarnpkg.com/>`_ package manager for Javascript
+- A terminal application for command line interaction
+
+
+Running the app
+***************
+
+We'll start by getting the app up and running, and then explain the different components which we will later extend.
+
+First off, open a terminal, clone the template project and move to the project folder::
+
+    git clone https://github.com/digital-asset/create-daml-app.git
+    cd create-daml-app
+
+In order to connect the DAML model to the UI code, we need to run a code generation step (more on this later)::
```

Comment:
We also compile the DAML models here. That's worth mentioning. Maybe something like
```suggestion
In order to connect the DAML model to the UI code, we need to compile the model and generate some TypeScript to interact with it (more on this later)::
```